tests/host_tools/fcmetrics.py

```diff
@@ -33,6 +33,12 @@ def create_metrics_schema_objects(metrics):
     }
 
     if isinstance(metrics, dict):
+        special_metrics = "utc_timestamp_ms"
```

Comment:
commit message `fix(test): Update metrics scheme to cover missing metrics`:

```diff
-added to the schema but, add it now to keep jsconschema.validate happy.
+added to the schema but, add it is now to keep jsonschema.validate happy.
```

Reply:
update:
commit message `fix(test): Update metrics scheme to cover missing metrics`:

```diff
-added to the schema but, add it now to keep jsconschema.validate happy.
+added to the schema but, add it now to keep jsonschema.validate happy.
```

Reply:
done, thanks!